WordPress/src/main/java/org/wordpress/android/ui/comments/CommentDetailFragment.java

```diff
@@ -136,7 +136,7 @@ public abstract class CommentDetailFragment extends ViewPagerFragment implements
     @Nullable private OnPostClickListener mOnPostClickListener;
     @Nullable private OnCommentActionListener mOnCommentActionListener;
     @Nullable private OnNoteCommentActionListener mOnNoteCommentActionListener;
-    @Nullable private CommentSource mCommentSource; // this will be non-null when onCreate()
+    protected CommentSource mCommentSource; // this will be non-null when onCreate()
```

Comment:
Wdyt of setting it as non-null?

```suggestion
    @NonNull protected CommentSource mCommentSource; // this will be non-null when onCreate()
```

Reply:
LGTM!